.gitignore

```diff
@@ -19,7 +19,7 @@ scripts/.openapi-generator
 api/.openapi-generator/*
 
 # config file
-src/instance/app.cfg
+src/hs_ontology_api/instance/app.cfg
```

Comment:
@ChuckKollar Due to the config directory change, you'll also need to update the docker mapping path: https://github.com/x-atlas-consortia/hs-ontology-api/blob/lifting_state_for_clearset_all_in_model/docker/docker-compose.yml#L28

Reply:
I made the fix